pkg/kubelet/server.go

```diff
@@ -641,7 +640,7 @@ func waitForPortForwardDataStreamAndRun(pod string, uid types.UID, errorStream h
 
 	select {
 	case dataStream = <-dataStreamChan:
-	case <-time.After(1 * time.Second):
+	case <-time.After(30 * time.Second):
```

Comment:
nit: perhaps replace with a constant, as in the client code? Since it appears two places.